.goreleaser.yml

```diff
@@ -29,5 +24,23 @@ checksum:
 sboms:
   - artifacts: archive
 
+release:
+  ids: [nginx-meshctl]
+  header: |
+    NGINX Service Mesh Release {{.Version}}
+
+    [Release Notes](https://docs.nginx.com/nginx-service-mesh/releases/release-notes-{{.Version}}/)
+
 archives:
-  - id: nginx-service-mesh
+  - id: nginx-meshctl
+    builds: [nginx-meshctl]
+    name_template: nginx-meshctl_{{ .Version }}_{{ .Os }}_{{ .Arch }}
+    format_overrides:
+      - goos: windows
+        format: zip
```

Comment:
Is this only saving the Windows binary?

Reply:
Or is this only for the purpose of generating the SBOM?

Reply:
By default, binaries are saved as `tar.gz`. This makes it so the windows binary is saved as `zip`.

Reply:
I added a screenshot to the description of what the release will look like.